ChaosMod/Effects/db/Vehs/VehsFaultyBrakes.cpp

```diff
@@ -0,0 +1,34 @@
+#include <stdafx.h>
+
+static void OnTick()
+{
+	static DWORD64 lastTick = 0;
+	DWORD64 curTick = GET_GAME_TIMER();
+	Ped playerPed = PLAYER_PED_ID();
+
+	if (lastTick < curTick - 2500)
+	{
+		lastTick = curTick;
+
+		if (IS_PED_IN_ANY_VEHICLE(playerPed, 0))
+		{
+			Vehicle veh = GET_VEHICLE_PED_IS_IN(playerPed, 0);

```

Comment:
you can move that inside the if below, no need to ask for the vehicle if we're not doing anything